charts/scalyr-agent/templates/kube-state-metrics/NOTICE.txt

```diff
@@ -0,0 +1,3 @@
+{{/*
```

Comment:
If you don't use syntax for the comment, helm will try it to apply is a Kubernetes resource and fail.

Reply:
Maybe we need to keep it outside of the `templates` directory?